labs/lab2/bigapp/Dockerfile.annotated

```diff
@@ -19,7 +19,7 @@ RUN yum -y update
 >>> cache in our intermediate cached image layer
 
 # Common Deps
-RUN yum -y install openssl procps-ng
```

Comment:
im just matching what's already in the actual Dockerfile... looks like it came out with these commits:
https://github.com/dustymabe/summit-2017-container-lab/commit/4f8eeccc8bd25134e5522f9b01e9af39b3570cb5 & https://github.com/dustymabe/summit-2017-container-lab/commit/ffb3b5ee989a3da8986e38068e2ea9b3d688797f